subworkflows/nf-core/fastq_bwa_mem_samblaster/tests/nextflow.config

```diff
@@ -0,0 +1,11 @@
+process {
+    withName: BWA_MEM {
+        ext.prefix = { "${meta.id}.on.${meta.ref_id}.bwa.mem" }
```

Comment:
Why these prefixes?

Reply:
Although these prefixes are not necessary here but in a pipeline they are as fastqs from each sample are mapped to each reference.